test/test_ops.py

```diff
@@ -381,17 +381,20 @@ def test_variant_consistency_jit(self, device, dtype, op):
         _requires_grad = op.supports_autograd and (dtype.is_floating_point or
                                                    op.supports_complex_autograd(torch.device(device).type))
         samples = op.sample_inputs(device, dtype, requires_grad=_requires_grad)
+        
+        # Acquires variants to test
+        func = op.get_op()
+        method = op.get_method()
+        variants = {
+            # TODO: inplace tests currently fail, fix and add inplace variant
+            'function': func, 'method': method,
+        }   
+        if op.has_fake_function:
+            variants = {'method': getattr(torch.Tensor, op.name)}
```

Comment:
What about acquiring the method using `op.get_method()` and if the op doesn't have a method skipping the test?